cmd/copy.go

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
you can remove the license boiler plate since we have a LICENSE file in the root

Also, remove commented out code thats unused or delete this file (even better) if unused.

Reply:
The copy command is the next thing I am going to start working on today.  I will clean out the rest.

Reply:
I cleaned out the license stuff.